addons/test_mail_full/static/tests/tours/portal_composer_actions_tour.js

```diff
@@ -0,0 +1,24 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("portal_composer_actions_tour_internal_user", {
+    steps: () => [
+        {
+            trigger:
+                "#chatterRoot:shadow .o-mail-Composer button[title='Insert a Canned response']",
+            run: "click",
+        },
+        {
+            trigger:
+                "#chatterRoot:shadow .o-mail-Chatter-top .o-mail-Composer-suggestion:contains(Hello, how may I help you?)",
+        },
+    ],
+});
```

Comment:
Could add intermediate step that there's `::` in composer, so that the understanding of `o-mail-Composer-suggestion` is clear just at looking at the steps.